receiver/hostmetricsreceiver/internal/scraper/processscraper/metadata.yaml

```diff
@@ -164,6 +164,14 @@ metrics:
       value_type: int
       aggregation: cumulative
       monotonic: false
+
+  process.handles:
+    enabled: false
+    description: Number of handles held by the process.
+    extended_documentation: This metric is only available on Windows.
+    unit: '{count}'
+    gauge:
```

Comment:
I believe this should be non-monotonic sum

Reply:
Most Windows users who need to know their handle count will check it in something like ProcExp or the performance counters and see the handle count at the current point in time. I don't think I understand why non-monotonic sum would be used here over gauge for that.

Reply:
I think I understand non-monotonic sums more now. I can make that change to conform with `process.open_file_descriptors`.

Reply:
Done in 44c7bab